app/views/quotes/show.html.erb

```diff
@@ -80,4 +134,35 @@
       </div>
     </div>
   </div>
-</div>
\ No newline at end of file
+</div>
+
+
+
+<script>
+  function initMap() {
```

Comment:
Define this function inside a javascript file and provide the lat, lng and the condition in the if statement as parameters @melissacgonzalez